### PR TITLE
Week 3- Business Site Project

### PR DESCRIPTION
**Pull request by Fatima-GR**
Here's my pull request for this week's project: Build a business site.

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,261 @@
...
+  transform: scale(1.1, 1.1);
+}
+
+/* ACCORDION SECTION */
+
+.accordion-section {
+  margin:20px 10px;
+  padding:30px;
+  background-color: rgb(145, 31, 39);
+  border-radius: 10% 20%;
```
Comment (JessicaNordahl) [inline code/style.css:146]: Cool that you experimented with different % on the two sections (accordion and the form section). That made a very nice effect for the viewer

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,261 @@
...
+  border: 1px solid rgb(235, 100, 100);
+}
+
+.fab { 
+  color:#fff;
+  padding:20px 10px;
+}
+
+
+/* MEDIA QUERIES */
```
Comment (JessicaNordahl) [inline code/style.css:213]: Your media queries are very good. I get the "hang of it" from looking at your work. 
You keep it simple and that is the key, I guess

**Review by JessicaNordahl — COMMENTED** [review]
Looks good Fatima!
Easy and coompliant structure of the html and css. 

I like that you made the cursor into a "pointer" . Nice effect.
The responsivenes is looking very professional and I am impressed by your work!

The only question that I have for you is why you used "text-decoration: none"

Great work Fatima!

File: code/index.html [hunk 3]
```diff
@@ -3,28 +3,165 @@
...
+        <h2 class="register-heading">Register with us!</h2>
+        <form action="htpp://htppbin.org/anything" method="POST" id="form">
+          <label for="fname" class="labels">First name:</label>
+          <input
+            type="text"
+            id="fname"
+            class="inputs"
+            name="fname"
+            placeholder="Name"
+            autocomplete="off"
```
[the remark this answers is not in the document]
Comment (Fatima-Gamero-Romero) [reply code/index.html:49]: yes, exactly! So it disables the autocomplete when the user is typing his/her name.

File: code/index.html [hunk 4]
```diff
@@ -3,28 +3,165 @@
...
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+          <input type="checkbox" id="terms" name="terms" value="agree" />
+          <label for="terms">
+            I agree to the <u>terms and conditions of Privacy Policy</u></label
+          ><br />
+          <input type="checkbox" id="age" name="age" value="agree" />
+          <label for="age"> I am 18 or older</label><br />
+          <button type="submit" class="button">Sign up</button>
```
[the remark this answers is not in the document]
Comment (Fatima-Gamero-Romero) [reply code/index.html:89]: Yes, I used the (<br>) as I could not find a better solution for making the label tags going to the next line.

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,261 @@
...
+  padding: 1rem;
+  font-size: 1.1rem;
+  cursor: pointer;
+  color: rgb(238, 185, 194);
+  text-decoration: none;
+  font-weight: bold;
+}
+
+a:hover {
+  transform: scale(1.2, 1.2);
```
[the remark this answers is not in the document]
Comment (Fatima-Gamero-Romero) [reply code/style.css:70]: Yes, one of my favorites effects 😊

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,261 @@
...
+}
+
+.logo-text {
+justify-content: center;
+font-size: 1.3rem;
+margin-top: 15px;
+color: rgb(235, 100, 100);
+}
+
+span {
```
[the remark this answers is not in the document]
Comment (Fatima-Gamero-Romero) [reply code/style.css:36]: Yes of course! The span is an inline element used for styling purposes as it is not semantic, it does not gives any information but is very useful when you want to style specific words. So if you look at the logo of my project, I used a p tag to write Gymtasy club and as I wanted to style only the letters 'tasy' with the color pink, I used the span tag, which allowed me to style this 4 letters. 

😊 
Look at this example and I think you will understand my explanation: 
https://www.w3schools.com/tags/tag_span.asp

**Pull request closed without merging**